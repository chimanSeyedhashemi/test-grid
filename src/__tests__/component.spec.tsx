import * as React from "react";
import {CattodGrid} from "..";
import {create} from "react-test-renderer";

test("Component should show 'red' text 'Hello World'", () => {
  const component = create(<CattodGrid  height="400px"/>);
  const testInstance = component.root;

  expect(testInstance.findByType(CattodGrid).props.height).toBe("400px");

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});