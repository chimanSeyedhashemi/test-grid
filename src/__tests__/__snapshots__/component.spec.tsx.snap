// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should show 'red' text 'Hello World' 1`] = `
<div
  className="ag-theme-bootstrap parent-style"
>
  <div
    className="catod-container"
    style={
      Object {
        "height": "50px",
      }
    }
  >
    <div
      id="center"
    >
      <div
        className="catod-grid-size"
        id="myGrid"
      >
        <div
          style={
            Object {
              "height": "100%",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
